File: baekjoon_all/10000+/boj_10807.cpp
// Solve 2022-06-20
// Update 2023-12-19

#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;

    int nums[100];

    for (int i = 0; i < n; i++) {
        cin >> nums[i];
    }

    int v;
    cin >> v;

    int cnt = 0;

    for (int i = 0; i < n; i++) {
        if (nums[i] == v) {
            cnt++;
        }
    }

    cout << cnt << '\n';

    return 0;
}

File: baekjoon_all/10000+/boj_10807.java
// Solve 2023-11-17
// Update 2023-12-19

// 백준에 제출할 때는 class 이름을 Main으로 설정해야 함.

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class boj_10807 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        StringTokenizer st = new StringTokenizer(br.readLine(), " ");
        int arr[] = new int[n];

        for (int i = 0; i < n; i++) {
            arr[i] = Integer.parseInt(st.nextToken());
        }

        int v = Integer.parseInt(br.readLine());
        br.close();
        int cnt = 0;

        for (int i = 0; i < n; i++) {
            if (arr[i] == v) {
                cnt++;
            }
        }

        System.out.println(cnt);
    }
}
